v1_0/tflite/README.md

```diff
@@ -0,0 +1,42 @@
+# MobileDet SSD
```

Comment:
This should also list the location of the other TFLite files for the 1.0 release (e.g. where is link for MobileBERT models(s) that are approved for v1.0

Reply:
@mcharleb I don't know the v1.0 version of MobileBert.